test/Main.purs

```diff
@@ -39,8 +40,37 @@ instance arbitraryArbPath ∷ QC.Arbitrary ArbPath where
 pathPart ∷ Gen.Gen String
 pathPart = Gen.suchThat QC.arbitrary (not <<< Str.null)
 
+parsePrintCheck :: forall a b. Path a b Sandboxed -> Maybe (Path a b Unsandboxed) -> QC.Result
+parsePrintCheck input parsed =
+  if parsed == Just (unsandbox input)
```

Comment:
I guess this check is not the best, as I got this error at some point:
```
checking `parse <<< print` for `RelFile``
97/100 test(s) passed.
/Users/safareli/dev/slamdata/purescript-pathy/output/Control.Monad.Eff.Exception/foreign.js:29
    throw e;
    ^

Error: Test 40 (seed 846546249) failed: 
`parse (print path) != Just path` for path: `(((((((((currentDir </> dir "냫笼嬋ఱ护⿼") </> dir "ꘓၷ两뱖쌜") </> dir "沌뢑朿ガ茧") </> dir "ﹱ捴ኧ嵹") </> dir "횽") </> dir "⤋") </> dir "큛रꃙ") </> dir "/솛鼒") </> file "냫笼嬋ఱ护⿼")` which was re-parsed into `(Just (((((((((currentDir </> dir "냫笼嬋ఱ护⿼") </> dir "ꘓၷ两뱖쌜") </> dir "沌뢑朿ガ茧") </> dir "ﹱ捴ኧ嵹") </> dir "횽") </> dir "⤋") </> dir "큛रꃙ") </> dir "솛鼒") </> file "냫笼嬋ఱ护⿼"))`
	Printed path: "./냫笼嬋ఱ护⿼/ꘓၷ两뱖쌜/沌뢑朿ガ茧/ﹱ捴ኧ嵹/횽/⤋/큛रꃙ/솛鼒/냫笼嬋ఱ护⿼"
	Printed path': `(Just "./냫笼嬋ఱ护⿼/ꘓၷ两뱖쌜/沌뢑朿ガ茧/ﹱ捴ኧ嵹/횽/⤋/큛रꃙ/솛鼒/냫笼嬋ఱ护⿼")`
```

if we move show-ed values side by side difference becomes more clear (scroll to right end)
```
(((((((((currentDir </> dir "냫笼嬋ఱ护⿼") </> dir "ꘓၷ两뱖쌜") </> dir "沌뢑朿ガ茧") </> dir "ﹱ捴ኧ嵹") </> dir "횽") </> dir "⤋") </> dir "큛रꃙ") </> dir "/솛鼒") </> file "냫笼嬋ఱ护⿼")
(((((((((currentDir </> dir "냫笼嬋ఱ护⿼") </> dir "ꘓၷ两뱖쌜") </> dir "沌뢑朿ガ茧") </> dir "ﹱ捴ኧ嵹") </> dir "횽") </> dir "⤋") </> dir "큛रꃙ") </> dir "솛鼒") </> file "냫笼嬋ఱ护⿼")
```

The different part:
```
... </> dir "/솛鼒") </> file "냫笼嬋ఱ护⿼")
... </> dir "솛鼒") </> file "냫笼嬋ఱ护⿼")
```

Reply:
Used alpha numeric generator in resent amend so this issue is not going to occur